# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../udev/no_restart.patch
# Copyright (C) 2006 - 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2008-06-12 08:24:30.000000000 +0300
+++ ./Makefile	2009-02-01 20:40:48.000000000 +0200
@@ -264,11 +264,6 @@
 	@extras="$(EXTRAS)"; for target in $$extras; do \
 		$(MAKE) -C $$target $@ || exit 1; \
 	done;
-ifndef DESTDIR
-	- killall udevd
-	- rm -rf $(udevdir)/.udev
-	- $(sbindir)/udevd --daemon
-endif
 .PHONY: install-bin
 
 uninstall-bin:
